CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ the detailed section referring to by linking pull requests or issues.
 #### Minor
```

Comment:
![grafik](https://github.com/sovity/edc-ui/assets/30114507/2a0ce1a3-324d-4587-83b3-cc66e281a6ca)
![grafik](https://github.com/sovity/edc-ui/assets/30114507/727f635d-ebcc-41bd-a7b4-e440e5c3aa20)
![grafik](https://github.com/sovity/edc-ui/assets/30114507/51fe3502-fee7-4f69-a118-bfc7127c6255)

the icons are different